utils/load_path.ml

```diff
@@ -174,3 +223,14 @@ let find_uncap fn =
   with Not_found ->
     let fn_uncap = String.uncapitalize_ascii fn in
     !auto_include_callback Dir.find_uncap fn_uncap
+
+let find_visible_uncap fn =
```

Comment:
This looks very similar to `find_visible`. Can we merge the shared logic? (maybe by passing `get_paths` and `files_uncap`)